src/components/tag/Tag.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { DEFAULT_TEXT_SIZE } from '../text/Text.constants';
+
+import PTag from './Tag.props';
+import TagRoot from './Tag.root';
+
+const Tag = ({ variant, size = DEFAULT_TEXT_SIZE, text, ...rest }: PTag): JSX.Element => {
+    const onClick = (): void => {
+        if (variant === 'highlight') {
+            // eslint-disable-next-line no-console
+            console.log('#### Mention clicked!');
```

Comment:
did you forget to remove that? :P

Reply:
I wonder if we need to keep it for feedback when design team checks the components? Just to show that 'something happens' :D not sure though

Reply:
You can still add that to the story